SwiftLeeds/Views/About/ContentTileView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  ContentTileView.swift
+//  SwiftLeeds
+//
+//  Created by Alex Logan on 01/07/2022.
+//
+
+import SwiftUI
+
+struct ContentTileView: View {
+    let title: String
+    let subTitle: String?
+    let imageURL: URL?
+
+    var placeholderColor: Color = .accentColor
+    var imageBackgroundColor: Color = .accentColor
+    var imageContentMode: ContentMode = .fill
+
+    let onTap: () -> ()
+
+    private let cornerRadius: CGFloat = 12
```

Comment:
Can this use the cellRadius, or that constant changed to be used for bother areas?